broker-cli/commands/identity.js

```diff
@@ -31,6 +31,6 @@ async function getIdentity (args, opts, logger) {
 module.exports = (program) => {
   program
     .command('id', 'Gets the Public Key of the Broker Daemon')
-    .option('--rpc-address [rpc-address]', 'Location of the RPC server to use.', validations.isHost)
+    .option('--rpc-address [rpc-address]', 'Location of the RPC server to use', validations.isHost)
```

Comment:
is it worthwhile to consolidate this option somewhere so that we have consistent language across the commands?

Reply:
I think that's a good idea. as a side note,why do we have an option to pass `rpc-address`. isn't `rpc-address` and `certs` a pair which will be setup at the time of installation of client & either both should be configurable or none.

Reply:
@mdasif the setup is a little rough. In the past (before the TLS requirement for cli), you could simply specific another broker address with `rpc-address`. It is still possible to have multiple certs and use this command, however we should probably add a `--rpc-cert` tag that allows the user to specify a cert.